helusers/settings.py

```diff
@@ -4,9 +4,11 @@
 
 _defaults = dict(
     # Accepted audience, the API Token must have this in its aud field
+    # Supports multiple audiences as a list
```

Comment:
These are default settings for internal use. I think instead of adding new comments, even the already existing ones could be removed.

Reply:
Removed the comments.